src/api/users.iced

```diff
@@ -70,6 +72,29 @@ users.handle_add_data = (req, resp) ->
   }
   req.couch.use('_users').atomic('base', 'do_action', user, action).pipe(resp)
 
+users.reactivate_user = (client, user, callback) ->
+  action = {
+    action: 'u+'
```

Comment:
Does reactivate have the same functionality as u+ or should this be a different action string?